test/e2e/networks/simple.toml

```diff
@@ -1,5 +1,27 @@
+shinitial_height = 10
```

Comment:
Can you reset this to how it was originally

Reply:
yeah, sorry for the slip.